src/Components/Aspire.StackExchange.Redis/Aspire.StackExchange.Redis.csproj

```diff
@@ -7,19 +7,27 @@
     <Description>A generic Redis® client that integrates with Aspire, including health checks, logging, and telemetry.</Description>
     <PackageIconFullPath>$(SharedDir)redis-cube-red_white-rgb.png</PackageIconFullPath>
     <NoWarn>$(NoWarn);SYSLIB1100;SYSLIB1101</NoWarn>
+    <IsAotCompatible>false</IsAotCompatible>
```

Comment:
```suggestion
```

The library was AOT compatible before this change. We shouldn't be making it incompatible. We need to remove whatever code is causing this. I believe you may have copied too much.

Reply:
I've pushed a change to revert this. I also opened https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1625 to port the fix to the underlying OTel code.

Reply:
I updated the code sync instructions to apply the PR code until this is released.